acme/readthedocs.org.requirements.txt

```diff
@@ -1,10 +1,10 @@
 # readthedocs.org gives no way to change the install command to "pip
-# install -e .[docs]" (that would in turn install documentation
+# install -e certbot[docs]" (that would in turn install documentation
```

Comment:
I think we should either change these comments back or correct them to `acme[docs]`, `certbot-apache[docs]`, etc. Only  Certbot's version of this file runs `pip install -e .[docs]`. I wonder if `.` was meant as a shorthand for the current package.

Reply:
Weird. My guess is actually that the text was copied from certbot's version, and no one changed it. I think it makes most sense to correct them to the actual package name.

Reply:
That being said, the latter actually isn't true -- the dns plugins did run `pip install -e .[docs]`. I'm still convinced of my explanation for why it's like this, and think that the correction is still accurate.